### DISCUSSION TITLE
Request: ファイルアップロード先にAzure Blob Storageを追加して欲しい

### DISCUSSION THREAD

**Discussion by sakazuki** [growilabs/growi#8231] [Ideas / Requests]
ファイルアップロード先に[Azure Blob Storage](https://azure.microsoft.com/ja-jp/products/storage/blobs/)（AWSのS3、GCPのGCSに相当するサービス）を追加して欲しい。

動くコードがありますので、取り込み検討が可能であればPR上げさせていただければと思います。
https://github.com/weseek/growi/compare/master...sakazuki:growi:feat/add-azure-blob-storage

設定説明テキストはこちらとなります。
https://github.com/weseek/growi-docs/compare/master...sakazuki:growi-docs:feat/add-azure-blob-storage

ご検討よろしくお願いいたします。

**Comment by yuki-takei** [comment]
@sakazuki 取り込みの検討可能です。PR を作っていただけると助かります。
docs の方の PR も大歓迎です！

**Comment by sakazuki** [comment]
#8233 をマージしていただきました。ありがとうございます！
